lib/oauth2.js

```diff
@@ -76,6 +76,9 @@ function parseProfile(body) {
     familyName: json.family_name,
     displayName: `${json.given_name} ${json.family_name}`,
     picture: json.picture,
+    emails: [{ value: json.email, verified: json.email_verified }],
+    photos: [{ value: json.picture }],
+    name: { givenName: json.given_name, familyName: json.family_name },
```

Comment:
can we include provider also?
```suggestion
    name: { givenName: json.given_name, familyName: json.family_name },
    provider: 'linkedin',
```